src/app/credential-issuer/middleware.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
 
       return handleBackendResponse(req, res, apiResponse?.data);
     } catch (error) {
-      error.csrfToken = req.csrfToken();
+      error.csrfToken = req.csrfToken(true);
```

Comment:
The `true` here makes the library generate a new token for each request (as the previous library did) instead of one token per user session.

Reply:
This is the invocation that I'm suspicious of - why is it only this error handler that does this, and what is going to read this property of the error anyway?

(If it's not doing anything, can we remove it?)

Reply:
Good point, I'm not sure why we even have CSRF protection here as this is a callback from the CRI so presumably won't have a CSRF token anyway.
This line was added in https://github.com/govuk-one-login/ipv-core-front/pull/634/files by @AmritSidhu , Amrit do you remember why this was needed?
Worth noting that the new version of this function below does not have this CSRF code

Reply:
I also can't see anywhere that the CSRF property of the error object is used.

Reply:
I want to say that it was added due to it complaining that it was missing at the time and it required it. But im afraid I really cant remember any specific reason

Reply:
Turns out that this has already been removed in the move to TypeScript so isn't relevant to this PR any more :)